modules/context/api.go

```diff
@@ -214,6 +214,10 @@ func (ctx *APIContext) RequireCSRF() {
 
 // CheckForOTP validates OTP
 func (ctx *APIContext) CheckForOTP() {
+	if skip, ok := ctx.Data["SkipLocalTwoFA"]; ok && skip.(bool) {
```

Comment:
why we do need this for api ?!?

Reply:
because the CheckForOTP is the 2FA of the API.